lib/breezy_pdf_lite/intercept/html.rb

```diff
@@ -8,12 +8,15 @@ def call
 
       render_request = BreezyPDFLite::RenderRequest.new(body).submit
 
+      expires = 10.minutes.from_now.utc.strftime("%a, %e %b %Y %H:%M:%S GMT")
```

Comment:
This uses ActiveSupport's date math monkey patching and wouldn't be non-rails compatible.